Userland/Libraries/LibWeb/UIEvents/MouseEvent.h

```diff
@@ -43,7 +43,7 @@ class MouseEvent final : public UIEvent {
 
     virtual u32 which() const override { return m_button + 1; }
 
-private:
+protected:
```

Comment:
I think only the constructor has to be protected, the private member function and private members can stay private.